### PR TITLE
Update Twitter logo and name

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -117,9 +117,9 @@
                 </a>
               </li>
               <li class="nav-item">
-                <a href="https://twitter.com/librepcb" title="Twitter"
+                <a href="https://twitter.com/librepcb" title="X"
                    class="nav-link icon icon-twitter">
-                  <i class="fa-brands fa-twitter fa-xl"></i>
+                  <i class="fa-brands fa-x-twitter fa-xl"></i>
```
Comment (ubruhin) [inline layouts/_default/baseof.html:122]: The icon `fa-x-twitter` is not contained in our icon set. Probably we need to update Fontawesome? Could you also take care of this?

Comment (WebSnke) [reply layouts/_default/baseof.html:122]: Yeah, sure. I will update the Font Awesome soon.

Comment (WebSnke) [reply layouts/_default/baseof.html:122]: I just updated the CSS for Font Awesome. Do I need to update anything else?

Comment (ubruhin) [reply layouts/_default/baseof.html:122]: The fonts in the "webfonts" directory need to be updated as well. I have now done that by myself and rebased this PR - sorry for the inconvenience.

Now the new logo works fine, thanks!

**Pull request merged**